kategory-core/src/main/kotlin/kategory/typeclasses/Bifoldable.kt

```diff
@@ -0,0 +1,16 @@
+package kategory
+
+interface Bifoldable<in F> : Typeclass {
```

Comment:
Contravariant `in` positions don;t work currently in the implicits system because java reflection does not see those how we need them to lookup instances by type so we should remove `in` positions from typeclasses.